gusgard/react-native-graphql-example
# Feature/components

* add Slider and ListGrid components
* add reselect, test and styles for components
* add travis config and fix test
* add border styles to grid
* upgrade RN version
* add styles for components
* add thumbnail, userBio components and parse bio
* refactor components and change styles
* add test for actions
* add missing test for component
* change eslint file to follow expected and add reselect test
* update readme and missing linter fixes
* add preview to readme
